lib/addressable/validator.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Addressable
+  ##
+  # Ensures that the URI is valid.
+  module Validator
+    METHODS_POOL = %w[
```

Comment:
I'm not sure `_POOL` is the right suffix to use for this identifier.

Reply:
yeah agree, thank you